demos/python_demos/llm_text_generation/servable_unary/model.py

```diff
@@ -184,5 +188,6 @@ def generate():
             else:
                 completions = [a + b for a, b in zip(completions, partial_result)]
         print('end', flush=True)
-
-        return serialize_completions(batch_size, completions)
+        t1.join()
+        completions.append(str(self.token_count))
```

Comment:
I don't think it is clean solution, it might show us that adding separate output is not bad idea. @dtrawins 
The same could be for stream endpoint. 
Suggestion - leave end_signal for what it is and add separate one.